src/conspiracies/corpusprocessing/umap_hdb.py

```diff
@@ -335,7 +343,7 @@ def embed_and_cluster(
 
     # Label and prune clusters
     print("Labeling clusters")
-    nlp = spacy.load("da_core_news_sm")
+    nlp = spacy.load("da_core_news_sm" if language == "danish" else "en_core_web_sm")
```

Comment:
again would just allow the user to specify the model

Reply:
A matter of preference. Configuration is good, but daunting unless you know the system. Configuring a language is pretty simply; configuring specific models less so.

But this is probably going away anyway in an upcoming PR.

Reply:
Yep, depends on the target audience - can def. see it both ways